exercises/J-array-get-set/exercises2.js

```diff
@@ -7,7 +7,8 @@
 */
 
 let numbers = [1, 2, 3]; // Don't change this array literal declaration
-
+ number[3] = 4;
```

Comment:
Hmm, what does number[3] points to? is it the last element?

Reminder: think about the starting index of array